Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -192,6 +233,135 @@ lemma rnDeriv_gaussianReal (μ : ℝ) (v : ℝ≥0) :
   · rw [gaussianReal_of_var_ne_zero _ hv]
     exact Measure.rnDeriv_withDensity _ (measurable_gaussianPdf μ v)
 
+section Transformations
+
+variable {μ : ℝ} {v : ℝ≥0}
+
+lemma _root_.MeasurableEmbedding.gaussianReal_comap_apply (hv : v ≠ 0)
+    {f : ℝ → ℝ} (hf : MeasurableEmbedding f)
+    {f' : ℝ → ℝ} (h_deriv : ∀ x, HasDerivAt f (f' x) x) {s : Set ℝ} (hs : MeasurableSet s) :
+    (gaussianReal μ v).comap f s
+      = ENNReal.ofReal (∫ x in s, |f' x| * gaussianPdfReal μ v (f x)) := by
+  rw [gaussianReal_of_var_ne_zero _ hv, gaussianPdf_def]
+  exact hf.withDensity_ofReal_comap_apply_eq_integral_abs_deriv_mul' hs h_deriv
+    (ae_of_all _ (gaussianPdfReal_nonneg _ _)) (integrable_gaussianPdfReal _ _)
+
+lemma _root_.MeasurableEquiv.gaussianReal_map_symm_apply (hv : v ≠ 0) (f : ℝ ≃ᵐ ℝ) {f' : ℝ → ℝ}
+    (h_deriv : ∀ x, HasDerivAt f (f' x) x) {s : Set ℝ} (hs : MeasurableSet s) :
+    (gaussianReal μ v).map f.symm s
+      = ENNReal.ofReal (∫ x in s, |f' x| * gaussianPdfReal μ v (f x)) := by
+  rw [gaussianReal_of_var_ne_zero _ hv, gaussianPdf_def]
+  exact f.withDensity_ofReal_map_symm_apply_eq_integral_abs_deriv_mul' hs h_deriv
+    (ae_of_all _ (gaussianPdfReal_nonneg _ _)) (integrable_gaussianPdfReal _ _)
+
+/-- The map of a Gaussian distribution by addition of a constant is a Gaussian. -/
+lemma gaussianReal_map_add_const (y : ℝ) :
+    (gaussianReal μ v).map (· + y) = gaussianReal (μ + y) v := by
+  by_cases hv : v = 0
+  · simp only [hv, ne_eq, not_true, gaussianReal_zero_var]
+    exact Measure.map_dirac (measurable_id'.add_const _) _
+  let e : ℝ ≃ᵐ ℝ :=
+  { toFun := fun x ↦ x - y
+    invFun := fun x ↦ x + y
+    left_inv := fun x ↦ by simp
+    right_inv := fun x ↦ by simp
+    measurable_toFun := measurable_id'.add_const _
+    measurable_invFun := measurable_id'.add_const _}
+  have he' : ∀ x, HasDerivAt e ((fun _ ↦ 1) x) x := fun _ ↦ (hasDerivAt_id _).sub_const y
+  suffices (gaussianReal μ v).map e.symm = gaussianReal (μ + y) v by exact this
```

Comment:
```suggestion
  change (gaussianReal μ v).map e.symm = gaussianReal (μ + y) v
```
If the goal is the same, use `change` instead of `suffices ... by exact this`.